build-system/compile/single-pass.js

```diff
@@ -255,10 +257,7 @@ exports.getBundleFlags = function(g) {
         jsFilesToWrap.push(name);
       } else {
         const configEntry = getExtensionBundleConfig(originalName);
-        const marker =
-          configEntry && Array.isArray(configEntry.postPrepend)
-            ? SPLIT_MARKER
-            : '';
+        const marker = configEntry ? SPLIT_MARKER : '';
```

Comment:
Every extension will now use a split marker for the intermediate modules. We'll add an additional one concatting the intermediates so that third_party code can be prepended.

Reply:
👍